housekeep.py

```diff
@@ -3,14 +3,30 @@
 import os
 import sys
 import pprint
+import inspect
 import argparse
 
 from pathlib import Path
 
+try:
+    # noinspection PyPackageRequirements
+    from colorama import (
```

Comment:
Something in the deps for testing pulls in `colorama`, so if it's available, let's leverage that to provide colorization on Windows cmd.